src/Lykke.Common.Validation/PasswordHash/PasswordHashFluentValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Text.RegularExpressions;
+using FluentValidation.Validators;
+
+namespace Lykke.Common.Validation.PasswordHash
+{
+    /// <summary>
+    ///     Validates string is a valid SHA-256 hash.
+    /// </summary>
+    public class PasswordHashFluentValidator : PropertyValidator
+    {
+        public PasswordHashFluentValidator() : base("{PropertyName} must have valid SHA-256 hash format.")
+        {
+        }
+
+        protected override bool IsValid(PropertyValidatorContext context)
+        {
+            var passwordHash = (string) context.PropertyValue;
+
+            return !string.IsNullOrWhiteSpace(passwordHash) && Regex.IsMatch(passwordHash, "^[A-Fa-f0-9]{64}$");
```

Comment:
Copy & past from https://github.com/LykkeCity/Lykke.Common.Validation/pull/1/files#diff-c13bdae984180808621d61d74deb86c8R20
Extract the method

Reply:
This is intended, as we will remove one approach and leave the second. 
We will use either Attribute or FluentValidator.